### ISSUE TITLE
'Editor' only refers to a type, but is being used as a value here.

### ISSUE THREAD

**Issue by yedidisatya** [telerik/kendo-vue#81]
![image](https://user-images.githubusercontent.com/31219029/95495083-854f4b00-09bc-11eb-96e7-40e86c123e8f.png)
While importing Editor form kendo-editor getting this issue in vue 3.0

**Comment by felipe-allmeida** [comment]
Hello! I'm having the same issue with the **multiselect** wrapper for vuejs.

Configuration:
    "@progress/kendo-dropdowns-vue-wrapper": "^2020.3.928",
    "@progress/kendo-ui": "^2020.3.915",
    "@progress/kendo-vue-dropdowns": "^1.1.2",
    "@progress/kendo-vue-intl": "^1.1.2",
    "bootstrap": "^4.5.2",

![image](https://user-images.githubusercontent.com/62676728/96479248-115c4080-120f-11eb-8f55-544420df1d1b.png)

**Comment by felipe-allmeida** [comment]
@yedidisatya I was able to solve the issue.

The wrapper implementation is slightly different for typescript codes, check their [documentation](https://www.telerik.com/kendo-vue-ui/components/framework/typescript-integration/).

I did the following to fix that issue:

**main.ts**
```ts
import '@progress/kendo-theme-bootstrap';
import '@progress/kendo-ui/js/kendo.multiselect' // Imports only the MultiSelect script and its dependencies
import { DropdownsInstaller } from "@progress/kendo-dropdowns-vue-wrapper";

Vue.use(DropdownsInstaller);
```

**TagSelect.vue**
```ts
<template>
    <kendo-multiselect ref="multiSelect"
        :data-source="items"
        :data-text-field="'text'"
        :data-value-field="'id'">
    </kendo-multiselect>
</template>

<script lang="ts">
import { MultiSelect } from '@progress/kendo-dropdowns-vue-wrapper';
import Vue from 'vue'
import { Prop } from 'vue-property-decorator';
export default class TagSelect extends Vue {
    @Prop({ required: true, default: [] }) items!: [];
    
    private multiSelectComponent!: kendo.ui.MultiSelect;

    mounted() {
        this.multiSelectComponent = (this.$refs.multiSelect as MultiSelect).kendoWidget() as kendo.ui.MultiSelect;
    }
}
</script>
```

**Comment by PekoPPT** [comment]
The same issue is reported for the ComboBox component in ticket #1492141. Attached you will find an example in which the issue can be replicated. 

To start the project -> npm Install -> npm run serve

**Comment by zdravkov** [comment]
The issue is fixed in the latest 2020.3.1118 versions of the components.
